pages/stable-version.html

```diff
@@ -181,13 +181,13 @@ <h5>Quickstart Installation Instructions</h5>
                 <code># transactional-update shell</code>
 
                 <p>5. Add the Uyuni Server repository:</p>
-                <code># zypper ar https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable/images/repo/Uyuni-Server-POOL-x86_64-Media1/ uyuni-server-stable</code>
+                <code># sudo zypper ar https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable/images/repo/Uyuni-Server-POOL-$(arch)-Media1/ uyuni-server-stable</code>
```

Comment:
```suggestion
                <code># zypper ar https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable/images/repo/Uyuni-Server-POOL-$(arch)-Media1/ uyuni-server-stable</code>
```

there's `#` at the beginning of the line, it should imply this requires root. or am I missing something?